doc/components/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ SET(LOCAL_FILES
 	pgr_biconnectedComponents.rst
 	pgr_articulationPoints.rst
 	pgr_bridges.rst
+    pgr_makeConnected.rst
```

Comment:
Is indention correct here?

Reply:
https://github.com/pgRouting/pgrouting/pull/1605/commits/0333207386c25011ab386dc22fcdb0b1a6569d21